### PR TITLE
Prepare snapcraft.yaml for publication into Snap

### PR DESCRIPTION
**Pull request by Stexxe**

### CODE REVIEW

File: snapcraft.yaml [hunk 1]
```diff
@@ -1,30 +1,42 @@
...
+version: 'dev'
+summary: Generates Ktor applications through the command line interface.
+website: https://github.com/ktorio/ktor-cli
+license: Apache-2.0
 description: |
-  Ktor is a web application framework for creating connected systems.
-  You can use it to create server-side as well as client-side applications.
-  It supports multiple platforms, including JVM, JavaScript, and Kotlin/Native.
-  This CLI tool provides functionality to create new Ktor projects.
+  Ktor is an asynchronous framework for creating microservices, web applications and more.
```
Comment (vnikolova) [inline snapcraft.yaml]:
```suggestion
  Ktor is an asynchronous framework for creating client and server applications, from microservices to multiplatform client apps.
```

File: snapcraft.yaml [hunk 2]
```diff
@@ -1,30 +1,42 @@
...
+  Written in Kotlin from the ground up.
+
```
Comment (vnikolova) [inline snapcraft.yaml:11]: Maybe we need to add here that this is the CLI tool for generating projects and specify what type of projects can be generated (client, server etc).

**Review by vnikolova — CHANGES_REQUESTED** [review]

File: snapcraft.yaml (hunk 2 above)
Comment (Stexxe) [reply snapcraft.yaml:11]: We have this in summary:
> Generates Ktor applications through the command line interface.

Should we add this to the description too?

Comment (vnikolova) [reply snapcraft.yaml:11]: In my understanding a summary is a summary of the description so it should be there, but I don't know how much details are necessary for this file specifically. What will the description be used for?

**Review by vnikolova — APPROVED** [review]

**Review by bjhham — APPROVED** [review]

**Pull request merged**